fastchat/model/model_adapter.py

```diff
@@ -624,6 +624,25 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("zero_shot")
 
 
+class BaichuanAdapter(BaseAdapter):
+    """The model adapter for baichuan-inc/baichuan-7B"""
+
+    def match(self, model_path: str):
+        return "baichuan" in model_path
+
+    def load_model(self, model_path: str, from_pretrained_kwargs: dict):
+        tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
+        config = AutoConfig.from_pretrained(model_path, trust_remote_code=True)
+        model = AutoModelForCausalLM.from_pretrained(
+            model_path, config=config, torch_dtype=torch.float16,
+            trust_remote_code=True, device_map="auto", low_cpu_mem_usage=True
+        )
+        return model, tokenizer
+
+    def get_default_conv_template(self, model_path: str) -> Conversation:
+        return get_conv_template("zero_shot")
```

Comment:
Does this template really work? Could you show a screenshot?

Reply:
![image](https://github.com/lm-sys/FastChat/assets/6822266/3f6a1923-03e0-4c53-a4dc-906437b8c079)
Ok, you can refer to the content in my screenshot.

Reply:
Is this model instruct fine-tuned? Will the `one_shot` template work better?

Reply:
According to the official statement from Baichuan, this is just a basic model that has not instruct fine-tuned. I also tried using the `one_shot` template and found that the results were similar.